javascript/asynchronous_javascript_and_apis/asynchronous_code.md

```diff
@@ -75,7 +75,9 @@ Of course, there are many more occasions where one would want to use Promises be
 1. Read this [Promises article](https://davidwalsh.name/promises). It's a good starting place and it's short and to the point.
 2. Watch this [Promises video](https://youtu.be/DHvZLI7Db8E).  It's a good place to get a feel for how one might actually use promises in the wild.
 3. Watch this [What is Event Loop? video](https://www.youtube.com/watch?v=8aGhZQkoFbQ) to understand how asynchronous code works in JavaScript.
-4. Read [promise basics](https://javascript.info/promise-basics), which is an excellent resource for beginners.
+4. Watch this [Event Loop - Visualized video](https://www.youtube.com/watch?v=8aGhZQkoFbQ) to further understand the Event Loop.
```

Comment:
It looks like this is a repeat of the previous video?

Reply:
Yeah they are just additional, repetitive videos. But thought the repetition and visualization were helpful in solidifying the knowledge. But yeah, can include in Additional resources instead, or if not necessary, then no need to include at all

Reply:
I think we might be talking past each other, I mean that it looks like your `Event Loop - Visualized` video is the same video that's linked under the `What is Event Loop?` link.

Reply:
Ohh I see, sorry my mistake. Have included the updated link in my second commit.

4. Watch this [Event Loop - Visualized video](https://www.youtube.com/watch?v=eiC58R16hb8) to further understand the Event Loop